lldb/include/lldb/Target/SwiftLanguageRuntime.h

```diff
@@ -234,7 +234,11 @@ class SwiftLanguageRuntime : public LanguageRuntime {
       const SymbolContext &sc,
       llvm::DenseMap<ArchetypePath, llvm::StringRef> &dict);
 
-  CompilerType DoArchetypeBindingForType(StackFrame &stack_frame,
+  /// Using the generic type parameters of \p stack_frame return a
+  /// version of \p base_type that replaces all generic type
+  /// parameters with bound generic types. If a generic type parameter
+  /// cannot be resolved, the input type is returned.
```

Comment:
nice.

should a future update change the return type to an optional or result type?

Reply:
I thought about that, but since the typical use is of the form `type = BindGenericTypeParameters(type)` that might make the call site more complex without much benefit.